db/migrate/20200623174413_hide_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class HideNotifications < ActiveRecord::Migration[6.0]
+  def change
+    add_column :notifications, :hidden, :boolean, default: false
```

Comment:
I don't know If I should also write data migration too?

Reply:
Adding this column will also provide ability to hide any notifications in the future.